lib/Controller/ScriptController.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+/**
+ * @copyright 2018 Christoph Wurst <christoph@winzerhof-wurst.at>
+ *
+ * @author 2018 Christoph Wurst <christoph@winzerhof-wurst.at>
+ *
+ * @license GNU AGPL version 3 or any later version
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+namespace OCA\Sentry\Controller;
+
+use OCP\AppFramework\Controller;
+use OCP\AppFramework\Http\DataDownloadResponse;
+use OCP\AppFramework\Http\Response;
+use OCP\AppFramework\Http\TemplateResponse;
+use OCP\IConfig;
+use OCP\IRequest;
+
+class ScriptController extends Controller {
+
+	/** @var IConfig */
+	private $config;
+
+	public function __construct(IRequest $request, IConfig $config) {
+		parent::__construct('sentry', $request);
+		$this->config = $config;
+	}
+
+	/**
+	 * @NoAdminRequired
+	 * @NoCSRFRequired
+	 */
+	public function dsn(): Response {
+		$pubDsn = $this->config->getSystemValue('sentry.public-dsn', null);
```

Comment:
This should be documented in the readme and maybe also add a link to https://docs.sentry.io/clients/javascript/usage/#preventing-abuse there.

Reply:
Good catch! I fixed it in the latest commit.